libs/sdk-ui-dashboard/src/presentation/filterBar/filterBar/FiltersConfigurationPanel.tsx

```diff
@@ -1,172 +1,152 @@
 // (C) 2021-2024 GoodData Corporation
 import React, { useState } from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+import { useTheme } from "@gooddata/sdk-ui-theme-provider";
+import { IAlignPoint, Icon, Typography, Bubble, BubbleHoverTrigger } from "@gooddata/sdk-ui-kit";
+
 import {
     selectDisableDashboardCrossFiltering,
     selectDisableDashboardUserFilterReset,
     selectDisableDashboardUserFilterSave,
+    selectDisableFilterViews,
     useDashboardDispatch,
     useDashboardSelector,
+    selectEnableFilterViews,
 } from "../../../model/index.js";
-
-import { IAlignPoint, Icon, Typography, Bubble, BubbleHoverTrigger } from "@gooddata/sdk-ui-kit";
 import { ConfigurationBubble } from "../../widget/common/configuration/ConfigurationBubble.js";
 import { metaActions } from "../../../model/store/meta/index.js";
-import { FormattedMessage } from "react-intl";
-import { useTheme } from "@gooddata/sdk-ui-theme-provider";
 
-const bubbleAlignPoints: IAlignPoint[] = [
-    { align: "tr tl", offset: { x: 0, y: 5 } },
-    { align: "tl tr", offset: { x: 0, y: 5 } },
-];
+const BUBBLE_ALIGN_POINTS: IAlignPoint[] = [{ align: "tc br", offset: { x: 4, y: -2 } }];
```

Comment:
There was huge blog of code cut&pasted for each configuration property. I extracted it to generic component and reused it for each toggle item.